flood_adapt/object_model/benefit.py

```diff
@@ -202,7 +202,7 @@ def cba(self):
         scenarios = self.scenarios.copy(deep=True)
         scenarios["EAD"] = None
 
-        results_path = self.database_input_path.parent.joinpath("output", "Scenarios")
+        results_path = self.database_input_path.parent.joinpath("output", "scenarios")
```

Comment:
This is the old way of getting paths. We have a function in the database controller to get the paths so at least all of the paths are constructed in the same way. 

see [dbsTemplate::get_database_path()](https://github.com/Deltares-research/FloodAdapt/blob/41a0a681aa6641ef856b5ebf8f0cf797cc21447b/flood_adapt/dbs_classes/dbs_template.py#L252)

The only problem is that we need to have the database accessible eveywhere, so its not a perfect solution by any means, but at least it more stable than hardcoding the strings and paths throughout the code